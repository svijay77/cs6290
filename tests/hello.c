#include <stdio.h>

int main(int argc, char *argv[]){
	printf("Hi! I am Sarvagnya Vijay!!!!\n");
	return 0;
}
