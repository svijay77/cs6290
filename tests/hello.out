Hi! I am Sarvagnya Vijay!!!!
